Split/Storage/Impressions/PersistentImpressionsCountStorage.swift

```diff
@@ -11,6 +11,48 @@ import Foundation
 protocol PersistentImpressionsCountStorage {
     func delete(_ counts: [ImpressionsCountPerFeature])
     func pop(count: Int) -> [ImpressionsCountPerFeature]
-    func push(counts: ImpressionsCountPerFeature)
+    func pushMany(counts: [ImpressionsCountPerFeature])
     func setActive(_ counts: [ImpressionsCountPerFeature])
 }
+
+class DefaultImpressionsCountStorage: PersistentImpressionsCountStorage {
+
+    private let impressionsCountDao: ImpressionsCountDao
+    private let expirationPeriod: Int64
+
+    init(database: SplitDatabase, expirationPeriod: Int64) {
+        self.impressionsCountDao = database.impressionsCountDao
+        self.expirationPeriod = expirationPeriod
+    }
+
+    func pop(count: Int) -> [ImpressionsCountPerFeature] {
+        let createdAt = Date().unixTimestamp() - self.expirationPeriod
+        let impressions = impressionsCountDao.getBy(createdAt: createdAt,
```

Comment:
you could rename `impressions` to `counts` or `impressionsCounts` for consistency.

Reply:
Done!